doc/classes/XRController3D.xml

```diff
@@ -7,7 +7,7 @@
 		This is a helper 3D node that is linked to the tracking of controllers. It also offers several handy passthroughs to the state of buttons and such on the controllers.
 		Controllers are linked by their ID. You can create controller nodes before the controllers are available. If your game always uses two controllers (one for each hand), you can predefine the controllers with ID 1 and 2; they will become active as soon as the controllers are identified. If you expect additional controllers to be used, you should react to the signals and add XRController3D nodes to your scene.
 		The position of the controller node is automatically updated by the [XRServer]. This makes this node ideal to add child nodes to visualize the controller.
-		As many XR runtimes now use a configurable action map all inputs are named. The name of the input is defined by the current [XRInterface]. In the case of OpenXR, these are the names of actions in the current action set.
+		As many XR runtimes now use a configurable action map, the current [XRInterface] defines the names of inputs. In the case of OpenXR, these are the names of actions in the current action set from the OpenXR action map.
```

Comment:
I still feel like it'd be better to drop the whole first half of the first sentence:

```suggestion
		The current [XRInterface] defines the names of inputs. In the case of OpenXR, these are the names of actions in the current action set from the OpenXR action map.
```

I don't think it really adds anything valuable